clients/runtime/src/tests.rs

```diff
@@ -79,49 +88,54 @@ async fn test_too_low_priority_matching() {
 	let (client, _tmp_dir) =
 		default_provider_client(AccountKeyring::Alice, is_public_network).await;
 	let parachain_rpc = setup_provider(client.clone(), AccountKeyring::Alice).await;
-	let err = parachain_rpc.get_too_low_priority_error(AccountKeyring::Bob.into()).await;
+	let err = parachain_rpc
+		.get_too_low_priority_error(AccountId(AccountKeyring::Bob.to_account_id().into()))
+		.await;
 	assert!(err.is_pool_too_low_priority())
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn test_subxt_processing_events_after_dispatch_error() {
-	let is_public_network = false;
-	let (client, _tmp_dir) =
-		default_provider_client(AccountKeyring::Alice, is_public_network).await;
-
-	let oracle_provider = setup_provider(client.clone(), AccountKeyring::Bob).await;
-	let invalid_oracle = setup_provider(client, AccountKeyring::Dave).await;
-
-	let key = primitives::oracle::Key::ExchangeRate(DEFAULT_TESTING_CURRENCY);
-	let converted_key = DiaOracleKeyConvertor::<MockValue>::convert(key.clone()).unwrap();
-	let exchange_rate = FixedU128::saturating_from_rational(1u128, 100u128);
-
-	let result = tokio::join!(
-		invalid_oracle.feed_values(vec![(converted_key.clone(), exchange_rate)]),
-		oracle_provider.feed_values(vec![(converted_key, exchange_rate)])
-	);
-
-	// ensure first set_exchange_rate failed and second succeeded.
-	result.0.unwrap_err();
-	result.1.unwrap();
-}
-
-#[tokio::test(flavor = "multi_thread")]
-async fn test_register_vault() {
-	let is_public_network = false;
-	let (client, _tmp_dir) =
-		default_provider_client(AccountKeyring::Alice, is_public_network).await;
-	let parachain_rpc = setup_provider(client.clone(), AccountKeyring::Alice).await;
-	set_exchange_rate(client.clone()).await;
-
-	let vault_id = VaultId::new(
-		AccountKeyring::Alice.into(),
-		DEFAULT_TESTING_CURRENCY,
-		DEFAULT_WRAPPED_CURRENCY,
-	);
-
-	parachain_rpc.register_public_key(dummy_public_key()).await.unwrap();
-	parachain_rpc.register_vault(&vault_id, 3 * 10u128.pow(12)).await.unwrap();
-	parachain_rpc.get_vault(&vault_id).await.unwrap();
-	assert_eq!(parachain_rpc.get_public_key().await.unwrap(), Some(dummy_public_key()));
-}
+// #[tokio::test(flavor = "multi_thread")]
```

Comment:
Instead of commenting them out, let's just add the [ignore](https://doc.rust-lang.org/reference/attributes/testing.html#the-ignore-attribute) macro, possibly also with an explanation that these tests are currently broken on Ubuntu and will be fixed later.